### ISSUE TITLE
Multiple possible roots

### ISSUE THREAD

**Issue by mfgumban** [marklogic-community/data-explorer#72]
When a document containing siblings of the same name, such as in:
```
<n:doc>
  <n:headers>
    <n:header>ABC</n:header>
    <n:header>DEF</n:header>
  </n:headers>
</n:doc>
```
is processed by the wizard, it displays multiple possible roots of the same path, in this case 2 instances of "/*:doc/*:headers/*:header".  In this case, should it only be one or is this intentional?

The fix seems nominal if its not intentional.

**Comment by markschiffner** [comment]
Yes, I agree we should fix that. I do not believe it was intentional.

**Comment by markschiffner** [comment]
Changed from question to a task
